bolt.gemspec

```diff
@@ -45,7 +45,7 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency "addressable", '~> 2.5'
   spec.add_dependency "aws-sdk-ec2", '~> 1'
-  spec.add_dependency "CFPropertyList", "~> 2.2"
+  spec.add_dependency "CFPropertyList", ">= 2.2"
```

Comment:
For posterity it looks like 3.0.6 is necessary for Ruby 3.2 support and the big breaking change between 2.x and 3.x was dropping support for Ruby 1.8 (and we complain about supporting 2.76!).